src/components/UserFilterDropdown/index.tsx

```diff
@@ -0,0 +1,123 @@
+import React, {useEffect, useState, ChangeEvent} from 'react';
+import {useDispatch} from 'react-redux';
+import {displayErrorToast} from 'utils/toasts';
+import {AppDispatch, SFC} from 'types';
+import * as S from './Styles';
+import axios from 'axios';
+import {UserReadSerializer} from 'types';
+import {authorizationHeaders} from 'utils/authentication';
```

Comment:
Let's keep up imports with the convention.

What I have understood is we have the following imports ordering:

1. React, Redux imports
2. External/third party libraries imports
3. Internal imports (sorted in ascending order)
4. Style import

So keeping this order in mind, we will have something like this:
```
import React, {useEffect, useState, ChangeEvent} from 'react';
import {useDispatch} from 'react-redux';
import axios from 'axios';

import {AppDispatch, SFC} from 'types';
import {UserReadSerializer} from 'types';
import {authorizationHeaders} from 'utils/authentication';
import {displayErrorToast} from 'utils/toasts';

import * as S from './Styles';
```